tests/conftest.py

```diff
@@ -1,32 +1,22 @@
+from __future__ import annotations
+
+import datetime
+import pathlib
+import typing as t
 from unittest.mock import patch
 
+import freezegun
 import globus_sdk
 import pytest
+import responses as responses_module
```

Comment:
I see this and the `responses` fixture below, but I would recommend simply naming the fixture something like `mocked_responses`. My reason for that is that it is sometimes necessary or useful for tests to still import `responses` and use it directly because some function is missing from `globus_sdk._testing`. If we allow the names to conflict, that will be slightly messier when/if needed.

Reply:
Great suggestion! I'll make this change.